deploy/kubernetes/driver/kubernetes/manifests/node-server.yaml

```diff
@@ -89,6 +89,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: SOCKET_PATH
+              value: "/tmp/mysocket.sock"
```

Comment:
In production based deployment socket should not be in tmp dir

Reply:
Updated the dir to /mhc-dir/mysocket.sock

The socket will be present in system /var/lib/ dir. Need to make changes to MHC repo as well for this.

Reply:
According to latest discussion, changed to "/var/lib/ibmshare.sock"